Makefile

```diff
@@ -0,0 +1,72 @@
+clean:
+	rm -rf bazel-*
+	rm -rf man
+	rm -rf docs
+
+build:
+	bazel build //...
+
+test:
+	bazel test //...
+
+.PHONY: clean build test
+
+bazel-bin/verilog/tools/lint/verilog_lint:
+	bazel build //verilog/tools/lint:verilog_lint
+
+bazel-bin/verilog/tools/syntax/verilog_syntax:
+	bazel build //verilog/tools/syntax:verilog_syntax
+
+bazel-bin/verilog/tools/formatter/verilog_format:
+	bazel build //verilog/tools/formatter:verilog_format
+
+tools: bazel-bin/verilog/tools/lint/verilog_lint bazel-bin/verilog/tools/syntax/verilog_syntax bazel-bin/verilog/tools/formatter/verilog_format
```

Comment:
I'd probably put this as the very first target. So that running `make` just results in these tools.